README.md

```diff
@@ -20,111 +20,52 @@ for more info).
 Issues in this repository are disabled for the same reason. Please, report all issues in the
 [crossterm-rs/crossterm](https://github.com/crossterm-rs/crossterm/issues) repository.
  
-## Table of contents:
-
-- [Getting started](#getting-started)
-- [Useful links](#useful-links)
-- [Features](#features)
-- [Examples](#examples)
-- [Tested Terminals](#tested-terminals)
-- [Authors](#authors)
-- [License](#license)
-
-## Getting Started
-
-All examples of how `crossterm_terminal` works can be found in the
-[examples](https://github.com/crossterm-rs/examples) repository.
-
-Add the `crossterm_terminal` package to your `Cargo.toml` file.
-
-```
-[dependencies]
-crossterm_terminal = "0.3"
-```
-
-And import the `crossterm_terminal` modules you want to use.
-
-```rust  
-pub use crossterm_terminal::{terminal, Terminal, ClearType};
-```
-
-### Useful Links
-
-- [Documentation](https://docs.rs/crossterm_terminal/)
-- [Crates.io](https://crates.io/crates/crossterm_terminal)
-- [Examples](https://github.com/crossterm-rs/examples)
-
 ## Features
 
-These are the features of this crate:
-
 - Cross-platform
 - Multi-threaded (send, sync)
 - Detailed Documentation
 - Few Dependencies
 - Terminal
-    - Clearing (all lines, current line, from cursor down and up, until new line)
-    - Scrolling (up, down)
-    - Terminal Size (get/set)
-    - Exit Current Process
-
-## Command API
-
-My first recommendation is to use the [command API](https://crossterm-rs.github.io/crossterm/docs/command.html)
-because this might replace some of the existing API in the future. It is more convenient, faster, and easier to use.
-
-## Examples
+  - Clear (all lines, current line, from cursor down and up, until new line)
+  - Scroll up, down
+  - Set/get the terminal size
+  - Exit current process
 
-The [examples](https://github.com/crossterm-rs/examples) repository has more complete and verbose examples.
+## Getting started
 
-```rust 
-use crossterm::terminal::{terminal,ClearType};
+<details>
+<summary>
+Click to show Cargo.toml.
+</summary>
 
-let mut terminal = terminal();
-
-// Clear all lines in terminal;
-terminal.clear(ClearType::All)?;
-// Clear all cells from current cursor position down.
-terminal.clear(ClearType::FromCursorDown)?;
-// Clear all cells from current cursor position down.
-terminal.clear(ClearType::FromCursorUp)?;
-// Clear current line cells.
-terminal.clear(ClearType::CurrentLine)?;
-// Clear all the cells until next line.
-terminal.clear(ClearType::UntilNewLine)?;
-
-// Get terminal size
-let (width, height) = terminal.size()?;
-print!("X: {}, y: {}", width, height);
-
-// Scroll down, up 10 lines.
-terminal.scroll_down(10)?;
-terminal.scroll_up(10)?;
+```toml
+[dependencies]
+# All crossterm features are enabled by default.
+crossterm = "0.11"
```

Comment:
crossterm_input?

Reply:
This is intentional. See the cursor's [Getting started](https://github.com/crossterm-rs/crossterm-cursor/blob/master/README.md#getting-started). Same here. Why intentional? Yep, it's a `crossterm_terminal` crate, but I kind of want to "force" users to use `crossterm` crate.